### PR TITLE
(sparkleshare) Embedded package

### PR DESCRIPTION
**Pull request by jack1142**
## Description
This makes the package sparkleshare embedded. I added legal folder with verification file and license and edited update script, so it embeds the packages.

## Motivation and Context
Package should be embedded if allowed and this app's license allows us to embed the installer (ref: #866)

## How Has this Been Tested?
I checked update.ps1, to see if it works properly, I tested the package in testing environment to see, if it installs and uninstalls properly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by jack1142** [comment]
I don't know if you check PRs with failed verification, so I just want to let you know that I think my PR (#1203) with python2 package might have failed the verification, because AppVeyor's Windows image already has python.

File: automatic/sparkleshare/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
   softwareName  = 'SparkleShare'
 
-  checksum      = '7c82078c4ed912d4ac5a0d93c9a638f15b9da5332f34d5088ec27ae14ccd085c'
-  checksumType  = 'sha256'
-
   silentArgs    = "/qn /norestart /l*v `"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
   validExitCodes= @(0)
 }
 
 Install-ChocolateyPackage @packageArgs
```
Comment (AdmiringWorm) [inline automatic/sparkleshare/tools/chocolateyInstall.ps1]: This call also needs to be changed
```suggestion
Install-ChocolateyInstallPackage @packageArgs
```

**Comment by jack1142** [comment]
I changed it. I didn't know there are actually two separate commands for installing local and remote files. This means, I may have used it in other PRs as well. I can check them and send a PR for these changes if you want.

**Comment by AdmiringWorm** [comment]
If you would be willing to do that, then sure. Feel free (you can do them in the same PR if you want).

It's very possible that I missed those calls when I reviewed the others.

**Comment by AdmiringWorm** [comment]
I was actually thinking of in a new PR, but just combine all the packages.

But sure, you can do it in this PR as well, I don't mind.

Just let me know when you're ready.

**Comment by jack1142** [comment]
I wanted to combine all the packages in one, new PR from the beginning, but if I can do it here as well, then I've just done it and it's ready.

**Pull request merged**